pytorch_lightning/accelerators/pipe_accelerator.py

```diff
@@ -0,0 +1,109 @@
+import os
+from typing import List
+
+import torch
+from fairscale.nn.pipe import microbatch
+from torch import nn
+
+import torch.distributed as torch_distrib
+from torch.distributed import rpc
+
+from pytorch_lightning import _logger as log
+from fairscale.nn import Pipe
+import fairscale.nn.model_parallel as mpu
+from fairscale.nn.pipe.pipeline import PipelineStyle
+from torch.nn.parallel import DistributedDataParallel
+
+from pytorch_lightning import LightningModule
+from pytorch_lightning.accelerators import DDPAccelerator
+from pytorch_lightning.plugins.ddp_plugin import DDPPlugin
+
+
+def get_worker_map():
+    return {rank: f"Test{rank}" for rank in range(torch_distrib.get_world_size())}
+
+
+class LightningPipeModule(nn.Module):
+    def __init__(self, module: nn.Sequential, layer_partitions: List[int], microbatches: int = 8):
+        super().__init__()
+        self.module = module
+        self.layer_partitions = layer_partitions
+        self.microbatches = microbatches
+
+    def init_pipe(self):
+        self.module = Pipe(
+            module=self.module,
+            balance=self.layer_partitions,
+            chunks=self.microbatches,
+            style=PipelineStyle.MultiProcess,
+            worker_map=get_worker_map()
+        )
+
+    def forward(self, *args, **kwargs):
+        x = self.module(*args, **kwargs)
+        if not self.training:
+            x = microbatch.gather(x)
```

Comment:
This doesn't seem correct, there needs to be some form of sync happening (hopefully answered by https://github.com/facebookresearch/fairscale/issues/173)